seed/search.py

```diff
@@ -64,6 +65,46 @@ def search_buildings(q, fieldnames=None, queryset=None):
     return queryset.filter(qgroup)
 
 
+def _search(q, fieldnames, queryset):
+    """returns a queryset for matching objects
+    :param str or unicode q: search string
+    :param list fieldnames: list of model fieldnames
+    :param queryset: "optional" queryset to filter from, will all return an empty queryset if missing.
+    :returns: :queryset: queryset of matching buildings
+    """
+    if q == '':
+        return queryset
+    qgroup = reduce(operator.or_, (
+        Q(**{fieldname + '__icontains': q}) for fieldname in fieldnames
+    ))
+    return queryset.filter(qgroup)
+
+
+def search_properties(q, fieldnames=None, queryset=None):
+    if queryset is None:
+        return PropertyState.objects.none()
+    if fieldnames is None:
+        fieldnames = [
+            'pm_parent_property_id'
+            'jurisdiction_property_identifier'
+            'address_line_1',
+            'property_name',
+        ]
+    return _search(q, fieldnames, queryset)
+
+
+def search_taxlots(q, fieldnames=None, queryset=None):
+    if queryset is None:
+        return TaxLotState.objects.none()
+    if fieldnames is None:
+        fieldnames = [
+            'jurisdiction_taxlot_identifier',
+            'address'
```

Comment:
I already merged this down, but the disconnect between the DB fields for tax lot and property is concerning. We call address_line_1 in one and address in the other. should we fix this? @RDmitchell